setup.py

```diff
@@ -26,15 +26,9 @@
 import subprocess
 import textwrap
 
-
-if sys.version_info[:2] < (3, 5):
-    raise RuntimeError("Python version >= 3.5 required.")
-
 import builtins
 
 
-python_requires = '>=3.5'
```

Comment:
This variable was not being used. An alternative is to do `python_requires=python_requires` below.

Reply:
deleting this seems right, no need for this variable